front-end/src/pages/Useronboardinga/Useronboardinga.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./Useronboardinga.css";
+import Usersnav from "../../component/molecule/Usersnav/Usersnav";
+import Logo from "../../../public/image/useronboard1.png";
+import { GrFormPrevious } from "react-icons/gr";
+import { GrFormNext } from "react-icons/gr";
+
+
+function Useronboardinga() {
+  return (
+    <>
+      <Usersnav />
+      <div className="useronboard_container">
+        <div className="div_elements">
+          <div className="header">
+            <h1>You are in</h1>
+          </div>
+          <div className="useronboarding_img">
+            <img src={Logo} alt="img" />
+          </div>
+          <div>
+            <p>
+              Are you ready to test your knowledge? This quiz app was <br />{" "}
```

Comment:
This text does not feets a teacher onboarding. Accomodate it for a teacher/user who sets the quizzes